src/streamlink/stream/ffmpegmux.py

```diff
@@ -181,7 +182,15 @@ def __init__(self, session, *streams, **options):
         copyts = session.options.get("ffmpeg-copyts") or options.pop("copyts", False)
         start_at_zero = session.options.get("ffmpeg-start-at-zero") or options.pop("start_at_zero", False)
 
+        if session.options.get("ffmpeg-verbose-path"):
+            loglevel = session.options.get("ffmpeg-loglevel") or options.pop("loglevel", self.DEFAULT_LOG_LEVEL)
+        elif session.options.get("ffmpeg-verbose"):
+            loglevel = session.options.get("ffmpeg-loglevel") or options.pop("loglevel", self.DEFAULT_LOG_LEVEL)
+        else:
+            loglevel = "quiet"
```

Comment:
(I think you'll understand if you look at it) I used the conditional judgment described below as is.

By the way, if it is not specified in "ffmpeg-verbose", I set it to subprocess.DEVNULL, but I think setting it to "quiet" would have the same effect. What do you think?
[L221](https://github.com/streamlink/streamlink/blob/11a6094c065879fe4c5e042f1ef2f7e3f0128c45/src/streamlink/stream/ffmpegmux.py#L221)

Reply:
The intention is to not introduce unnecessary code branches that would need to be covered by tests. It doesn't matter which log level is set on the FFmpeg process if the stderr stream is written to devnull anyway. It's a miniscule performance improvement.

The description of `--ffmpeg-verbose` already implies that the default info log won't be written to streamlink's overall stderr output, but this could be improved by adding a note to the `--ffmpeg-loglevel` help text.